base_layer/wallet_ffi/tests/test_network_exploration.py

```
@@ -0,0 +1,463 @@
+
+import pytest
+import os
+from pathlib import Path
+
+# Set environment for nextnet testing
+os.environ['TARI_TARGET_NETWORK'] = 'nextnet'
+
+# Add Python module path
+current_dir = Path(__file__).parent
+python_module_path = current_dir.parent / 'python'
+import sys
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)